jdaviz/utils.py

```diff
@@ -385,3 +391,117 @@ def total_masked_first_data(self):
     def __setgluestate__(cls, rec, context):
         masks = {key: context.object(value) for key, value in rec['masks'].items()}
         return cls(masks=masks)
+
+
+def download_uri_to_path(possible_uri, cache=None, local_path=os.curdir):
```

Comment:
Since the mention of standalone app this morning, I wonder now if `os.curdir` is okay in standalone mode. Don't we want something that is consistent with Export plugin behavior?

Reply:
My take is: if you've run standalone from your terminal, you might expect that files get saved to the path where you launched jdaviz.

Reply:
Right, that is the expectation, but working dir is actually some `/tmp` dir the voila server is running from. You would need something like this to access the path where you launched Jdaviz via CLI.

https://github.com/spacetelescope/jdaviz/blob/50ebc1ebbfe24f9e3e5c01c87d8e7e0d3772a36d/jdaviz/configs/default/plugins/export/export.py#L349-L350

Reply:
Ah, I didn't realize this, thanks. (and don't use standalone enough to have found this out naturally 🙃)